src/styles/components/decorated-list-item.scss

```
@@ -97,6 +131,7 @@ $alt-decorative-bar: 0.25rem solid $colour-pastel-blue;
     &__content {
       border-left: none;
       padding-left: 0;
+      padding-left: 0;
```

```suggestion
      padding-left: 0;
```

Might be unwanted I believe